src/DetailsView/components/narrow-mode-thresholds.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export type NarrowModeThresholds = {
+    collapseHeaderAndNavThreshold: number;
+    collapseCommandBarThreshold: number;
+};
+
+export const getNarrowModeThresholdsForWeb = (): NarrowModeThresholds => {
+    return {
+        collapseHeaderAndNavThreshold: 600,
+        collapseCommandBarThreshold: 960,
+    };
+};
+
+export const getNarrowModeThresholdsForUnified = (): NarrowModeThresholds => {
```

Comment:
nit: Might not be a bad idea to have this in separate file specifically inside the electron/unified file structure rather than the DetailsView.